aries-site/src/layouts/content/UpdateTag.js

```diff
@@ -0,0 +1,42 @@
+import { Anchor, Text, Notification } from 'grommet';
+import { CircleInformation } from 'grommet-icons';
+import { ViewContext } from '../../pages/_app';
+import { useContext } from 'react';
+
+export const UpdateTag = ({name}) => {
+    const {wholeViewHistory} = useContext(ViewContext) || undefined;
+        if(wholeViewHistory && name in wholeViewHistory){
+            return (
+                <>
+                {(wholeViewHistory[name].type === "Update") &&
+                    <Notification width='large'
+                    title={"Updated " + new Date(wholeViewHistory[name].date).toDateString()} 
+                    message={
+                        <>
+                        <Text>{wholeViewHistory[name].description + "    "}
+                        {wholeViewHistory[name].action.length > 1 && 
+                            <Anchor label='Link' href={wholeViewHistory[name].action}/> 
+                        }
+                        </Text>
+                        </>
```

Comment:
```suggestion
                        <Text>{wholeViewHistory[name].description + "    "}
                        {wholeViewHistory[name].action.length > 1 && 
                            <Anchor label='Link' href={wholeViewHistory[name].action}/> 
                        }
                        </Text>
```
Shouldn't need a fragment here